nf_core/__main__.py

```diff
@@ -389,6 +340,394 @@ def lint_pipeline(
         sys.exit(1)
 
 
+# nf-core pipelines download
+def pipelines_download(
```

Comment:
Maybe stupid question, but why are these commands defined above pipelines()? couldn't we just define them with the actual commands?

Reply:
They have to be separate functions because if we use the command directly, it uses the decorated function, so there are problems with the parameters and the context. I found [different solutions](https://stackoverflow.com/questions/40091347/call-another-click-command-from-a-click-command), but the best seems to be to have separate functions.
Grouping all of them at the top of the file is just a personal preference 😄

Reply:
would it make sense to move these out of __main__.py to make that file smaller?

Reply:
yes, I like this option! But do we prefer more and smaller files? or less files even if it's too big? We could also move the functions for modules and subworkflows if we want

Reply:
More and smaller files 👍🏻  Prefer having a `cli.py` or similar just for the command line wrapper functions..

Reply:
I separated the functions into different files with https://github.com/nf-core/tools/pull/3039/commits/9b965e1981309e46b336fdb927b321c702dbc536